common/changes/office-ui-fabric-react/codepen-example-data_2019-01-15-01-15.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Update DetailsList examples to have codepens",
+      "type": "patch"
```

Comment:
Just a quick question: if the changes in OUFR didn't touch any of the code that is going to be released... do we still have the type `patch` instead of `none` here?

Reply:
Hmm I'm actually not sure what the proper thing is to do in this case--I went with patch because the website consumes the examples, but that could be wrong.

Reply:
Isn't the website a different package? @Jahnp do you have any context in here and let us know how is the website picking up changes in the examples we have for each component? It appears that patching the OUFR package should not do it... but I might be wrong.

Reply:
I see small modifications to .doc files, so I think it probably makes sense to patch.

Reply:
@ecraig12345 is right--the website is tied to OUIFR's releases, so patch bumps ensure change shows up on site.

Reply:
Are the `.doc` files part of the release?